homeassistant/components/philips_js/config_flow.py

```diff
@@ -53,34 +67,97 @@ async def async_step_import(self, conf: Dict[str, Any]):
             }
         )
 
+    async def _async_create_current(self):
+
+        system = self._current[CONF_SYSTEM]
+        return self.async_create_entry(
+            title=f"{system['name']} ({system.get('serialnumber', )})",
```

Comment:
Since serial number can be None on a v6 system, should we only append the parentheses if there is a serial?

```suggestion
            title=f"{system['name']} ({system.get('serialnumber')})",
```

Reply:
This was actually an invalid assumption before the library was able to decode the encrypted serial number. So i can just assume it exist now.